lib/async_hooks.js

```diff
@@ -321,10 +334,16 @@ function setInitTriggerId(triggerAsyncId) {
 
 
 function emitInitScript(asyncId, type, triggerAsyncId, resource) {
+  validateAsyncId(asyncId, 'asyncId');
+  if (triggerAsyncId !== null)
+    validateAsyncId(triggerAsyncId, 'triggerAsyncId');
+  if (async_hook_fields[kSanityCheck] > 0 &&
+     (typeof type !== 'string' || type.length <= 0)) {
```

Comment:
Indentation is missing a single space here?

Reply:
I don't know. The linter says it is fine.

Reply:
hm. another one of those things we do so regularly I didn't realize we needed a linter fix for. If you run this you'll see that >90% of these would have one more space
```
git grep -A 1 " if (.*[^{);]$" lib/
```
i'll check on getting a rule into the linter.